rtgui/inspector.cc

```diff
@@ -166,6 +171,25 @@ bool Inspector::on_button_press_event(GdkEventButton *event)
     return false;
 }
 
+bool Inspector::on_motion_notify_event(GdkEventMotion *event)
+{
```

Comment:
Make sure `currImage` is not null here (and also in `Inspector::on_zoom_begin`, I think).

Reply:
Thank you for the code review. Added two checks to 947f3dc.